rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -92,7 +92,7 @@ setup:
   - is_true: test_index.aliases
   - is_true: test_index.settings
   - is_true: test_index_2.settings
-  - is_true: test_index_2.mappings
+  - is_false: test_index_2.mappings
```

Comment:
This started failing after I made the changes you requested to `GetIndexResponse#toXContent`. Not sure what the expected behaviour should be here, the index doesn't contain any user-specified mapping but it looks like we printed an empty one in the response so far. Change the test or update the `toXContent` rendering, wdyt?

Reply:
It seems like it's simplest to change the `toXContent` to match the old behavior, as this will avoid BWC concerns.

Reply:
done